packages/testing/addon/mirage-graphql/schema.graphql

```diff
@@ -89,6 +100,240 @@ type AddWorkflowFlowPayload {
   clientMutationId: String
 }
 
+"""
+Aggregate function for pivot table
+"""
+enum AggregateFunction {
+  VALUE
+  SUM
+  COUNT
+  AVG
+  MAX
+  MIN
+}
+
+"""
+A cell represents one value in the analytics output.
+"""
+type AnalyticsCell {
+  alias: String
```

Comment:
For what is the `alias` on the different types?

Reply:
This is just a property named alias.